### DISCUSSION TITLE
PAGImage不能支持webp动图

### DISCUSSION THREAD

**Discussion by qwhuang** [Tencent/libpag#1628] [General] [answered]
[PAGImage FromPath:]得到的是一个有宽高的PAGImage，但内容是透明的。能否返回第一帧图片？

**Comment by kevingpqi123** [comment, marked as answer]
对于动图暂时没有计划去支持，业务侧在做占位图替换前可以先做判断及处理，这部分内容放在外部也可以实现

**Reply by qwhuang** [reply]
谢谢，明白了。已经按您的办法解决了。
